modules/nw-multinetwork-sriov.adoc

```diff
@@ -312,3 +468,58 @@ $ oc create -f sriovsamplepod.yaml
 ----
 $ oc exec sriovsamplepod -- ip a
 ----
+
+User can also run RDMA or DPDK application in a pod with SR-IOV VF attached. The
+following is an example of a pod with a VF in RDMA mode:
+
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: rdma-app
+  annotations:
+    k8s.v1.cni.cncf.io/networks: sriov-rdma-mlnx
```

Comment:
Since we are giving an pod example here, might be good to also include the example of net-attach-def CR(`sriov-rdma-mlnx`) used here.  This applies to dpdk example below as well.